scripts/jenkins/Jenkinsfile

```diff
@@ -182,11 +180,9 @@ pipeline {
         }
         failure {
             script {
-                if (!env.BRANCH_NAME.startsWith("PR-")) {
                     // Copy over any UI failure screenshots and send to slack
```

Comment:
Can you move the lines within the removed 'if' back a tab to keep proper spacing?